src/test/java/fshop/db/FoodDaoImplTest.java

```diff
@@ -0,0 +1,138 @@
+package fshop.db;
+
+import fshop.model.Food;
+import fshop.service.workfile.ReadCsv;
+import fshop.service.workfile.ReadCsvImpl;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class FoodDaoImplTest {
+    @Test
+    void createObject_null() {
```

Comment:
add access modifiers to all test classes and their methods

Reply:
done